packages/s3-request-presigner/README.md

```diff
@@ -91,4 +91,46 @@ to `presigned` is not sufficient to make a request. You need to send the
 server-side encryption headers along with the url. These headers remain in the
 `presigned.headers`
 
+getSignedUrl Example:
```

Comment:
Add two sections

```md
### Get Presigned URL with headers that cannot be signed

Description, and reason

Example

### Get Presigned URL with headers that should be signed

Description, and reason

Example
```

Reply:
Thank you, hopefully the changes are better :)